ctpolicy/ctpolicy.go

```diff
@@ -79,7 +85,15 @@ func (ctp *CTPolicy) race(ctx context.Context, cert core.CertDER, group cmd.CTGr
 	// so we maximize the distribution of logs we get SCTs from.
 	for _, i := range rand.Perm(len(group.Logs)) {
 		l := group.Logs[i]
-		go func(l cmd.LogDescription) {
+		go func(i int, l cmd.LogDescription) {
+			// Each submission waits 500ms longer than the previous one, to give the
```

Comment:
I think the 500ms part of this comment isn't necessarily accurate since `group.Stagger.Duration` is configurable.